crates/starknet-server/test_data/cairo1/balance_contract/balance_contract.cairo

```diff
@@ -0,0 +1,48 @@
+#[starknet::interface]
+trait IBalanceContract<TContractState> {
+   fn increase_balance(ref self: TContractState, amount1: u128, amount2: u128);
```

Comment:
Do we need increase_balance and get_balance in this contract I have seen that only create_panic is called?

Reply:
I minimized the contract used and renamed it accordingly: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/247/commits/f4d199d4b6e74a6c29ffce1ece470a89e47ee26b